packages/cli/src/oclif/commands/push.js

```diff
@@ -12,7 +12,15 @@ class PushCommand extends ZapierBaseCommand {
   async perform() {
     const skipNpmInstall = this.flags['skip-npm-install'];
     const definition = await localAppCommand({ command: 'definition' });
-    const version = definition.version;
+
+    const snapshotLabel = this.flags.snapshot;
+    if (snapshotLabel && snapshotLabel.length >= 18) {
```

Comment:
18 is fine, and also matches the error.

```suggestion
    if (snapshotLabel && snapshotLabel.length > 18) {
```